Protos/V1/tasks_common.proto

```diff
@@ -184,6 +184,79 @@ message CancelTasksResponse {
   repeated TaskSummary tasks = 1;
 }
 
+/**
+ * Request to count tasks.
+ */
+message CountTasksRequest {
+  /**
+   * Represents every availaible field to use in order to group tasks.
+   */
+  enum GroupByField {
+    GROUP_BY_FIELD_UNSPECIFIED = 0; /** Unspecified. */
+    GROUP_BY_FIELD_STATUS = 1; /** The task status. */
+    GROUP_BY_FIELD_SESSION_ID = 2; /** The session ID. */
+    GROUP_BY_FIELD_APPLICATION_NAME = 3; /** The application name. */
+    GROUP_BY_FIELD_APPLICATION_VERSION = 4; /** The application version. */
+  }
+
+  /**
+   * The group by.
+   *
+   * Can't be empty. Can group on multiple fields at the same time.
+   */
+  repeated GroupByField group_by = 1;
+
+  /**
+   * Represents a filter with every available fields.
+   *
+   * Every field can be used as the same time.
+   */
+  message Filter {
```

Comment:
I am pretty sure Filter is not specific to `CountTasks`.

Reply:
Pour le moment, ce n'est pas exactement le même. Mais à voir à l'usage effectivement.